docs/src/components/Tabs.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import PropType from 'prop-types';
+
+export const TabItem = (props) => {
+	return props.children;
+};
+TabItem.propTypes = {
+	children: PropType.oneOfType([PropType.string, PropType.array]),
+};
+
+export const Tabs = (props) => {
+	const data = props.children || [];
+	const [visibleTab, setVisibleTab] = useState(0);
+	const listTitles = data.map((item, index) => (
+		<li key={`tab_${item.props.title}`}>
+			<button
+				role="tab"
+				id={`tab_${item.props.title}`}
+				aria-controls={`tabpanel_${item.props.title}`}
+				onClick={() => setVisibleTab(index)}
+				aria-selected={visibleTab === index ? 'true' : 'false'}
+				tabIndex={visibleTab === index ? '-1' : null}
+				className={
```

Comment:
no need to add --active modifier as styling should be keyed off of aria-selected